tests/fixtures/colortest.css

```diff
@@ -26,3 +26,8 @@
 	background-color: rgba(var(--some-rg), 255, 0.1);
 	background-color: hsla(var(--some-hsl), 0.1);
 }
+
+#css4-rgba {
+	background-color: rgb(242 245 249 / 45%);
```

Comment:
It would be cool to have an output format option to prefer this syntax over the other. AFAICT there are three boolean options that are all independent of one another: prefer-rgb-over-rgba / rgb-alpha-separator-use-slash / rgb-alpha-use-percentage.

Reply:
It would, though can (and should) be done as a separate PR.